README.md

```diff
@@ -27,20 +46,20 @@ Retro chiptune music is starting to make a resurgence with the popularity of man
 ---
 Mainly Rust, and possibly Python as well
 
-## Possible Function Implemenations
+## Expected Functions:
 ---
 * MIDI Input
 * Visualization (waveform)
 * GUI
 * Pass songsusing import 
 
-## Good to have:
+## Potential Functions:
 ---
 * Filters?
 * Entrie chip (real code)
```

Comment:
Dunno if that's supposed to be "Entry chip" or if how that is spelled

Reply:
Yeah not sure about that either tbh, just gonna leave it as is